modules/utility.js

```diff
@@ -142,43 +144,31 @@ function checkPermission(msg, guildMember) {
 
 async function getChannelData(msg, args) {
     try {
-        let channelByID = args;
-        let channelID;
-        if (args === undefined) {
-            channelID = msg.channel.id;
-        } else {
-            channelID = channelByID;
-        }
-        let channel;
-        try {
-            channel = await msg.guild.channels.cache.get(channelID);
-        } catch (error) {
-            console.error();
-        }
-        return channel;
+        const channels = msg.guild.channels;
+        let channelId = msg.channel.id;
+
+        if (args) channelId = args;
+        return await channels.fetch(channelId);
     } catch (error) {
         console.error(error);
+        return false;
     }
 }
 
 async function getRoleData(msg, args) {
     try {
-        let mention = msg.mentions.roles.first();
-        let roleID;
-        if (!mention) {
-            roleID = args;
-        } else {
-            roleID = mention.id;
-        }
-        let role;
-        try {
-            role = msg.guild.roles.cache.get(roleID);
-        } catch (error) {
-            console.error(error);
-        }
-        return role;
+        const mentions = msg.mentions;
+        const roles = msg.guild.roles;
+
+        const mention = mentions.roles.first();
+        let roleId = mention.id;
+
+        if (!mention) roleId = args;
+
+        return roles.fetch(roleId);
```

Comment:
Jika menggunakan code ini dan ingin mendapatkan info role melalu ID, maka akan menimbulkan error seperti gambar di bawah ini.
![Screenshot_1260](https://user-images.githubusercontent.com/87353624/149658551-ae863ef7-a51d-45c8-8a46-a0386b624358.png)
![Screenshot_1261](https://user-images.githubusercontent.com/87353624/149658567-fb11ddb8-cdcb-46e4-9184-cbe18c9c4bd6.png)
Hal tersebut dikarenakan code `let roleId = mention.id;` berada di atas pengecekan `if (!mention)`.

Saran:
```suggestion
        const mention = msg.mentions.roles.first();
        let roleId;

        if (!mention) {
        roleId = args;
        } else {
        roleId = mention.id;
        }
        return msg.guild.roles.fetch(roleId);
```